[2025-09-06T14:37:18.476+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [queued]>
[2025-09-06T14:37:18.492+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [queued]>
[2025-09-06T14:37:18.493+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 4
[2025-09-06T14:37:18.523+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_sql_db> on 2025-09-05 00:11:34.425860+00:00
[2025-09-06T14:37:18.539+0000] {standard_task_runner.py:60} INFO - Started process 652 to run task
[2025-09-06T14:37:18.545+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'banvic_etl', 'extract_sql_db', 'manual__2025-09-05T00:11:34.425860+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/banvic_dag.py', '--cfg-path', '/tmp/tmp2ib30423']
[2025-09-06T14:37:18.552+0000] {standard_task_runner.py:88} INFO - Job 76: Subtask extract_sql_db
[2025-09-06T14:37:18.795+0000] {task_command.py:423} INFO - Running <TaskInstance: banvic_etl.extract_sql_db manual__2025-09-05T00:11:34.425860+00:00 [running]> on host 90038f871efe
[2025-09-06T14:37:19.129+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='banvic_etl' AIRFLOW_CTX_TASK_ID='extract_sql_db' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T00:11:34.425860+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-05T00:11:34.425860+00:00'
[2025-09-06T14:37:19.255+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:19.444+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:19.498+0000] {logging_mixin.py:188} WARNING - /opt/airflow/scripts/extract_helpers.py:33 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-06T14:37:19.614+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-06T14:37:19.667+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=banvic_etl, task_id=extract_sql_db, execution_date=20250905T001134, start_date=20250906T143718, end_date=20250906T143719
[2025-09-06T14:37:19.724+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-06T14:37:19.806+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
